doc/about/api/index.md

```diff
@@ -74,12 +75,16 @@ a different (updated) on-disk format.
 # Platform stability {#platform_stability}
 
 The main requirement for being able to use SeqAn is a supported compiler and in general all operating systems that
-provide a supported compiler should be supported. We currently support the
-following compilers on 64-bit operating systems with little-endian CPU
-architectures only:
+provide a supported compiler should be supported. In general, we only support the latest three major compiler versions.
+
+We currently support the following compilers on 64-bit operating systems with little-endian CPU architectures:
   * GCC7, GCC8, GCC9, GCC10, GCC11
 
-We promise to support all of the above compilers until the last release of SeqAn3, or until all of the following
+\note Only the most recent minor release of a compiler is guaranteed to be supported, e.g. when `gcc-10.4` is released,
+we may drop support for `gcc-10.3`. Since all platforms with an older version receive minor release updates,
+this should not be a problem.
+
+We promise to support all the above compilers until the last release of SeqAn3, or until all the following
```

Comment:
Nicht ganz sicher was damit ausgedrückt werden soll, vielleicht so?:
```suggestion
We promise to support the above compilers in the latest release of SeqAn3, or until all the following
```

Reply:
I think it wants to express `Until SeqAn 4.0.0`. Though that's probably not correct anyway :)